environment_forge_full.yml

```diff
@@ -0,0 +1,41 @@
+name: qcodesforge
+channels:
+ - conda-forge
+dependencies:
+ - python=3.7.9
+ - pip
+ - deepdiff
+ - numpy=1.19.5
+ - matplotlib=3.3.2
+ - pyqtgraph=0.11.0
+ - h5py=2.10.0
+ - pyqt=5.9.2
+ - QtPy
+ - jsonschema
+ - jupyter=1.0.0
+ - hypothesis=6.0.0
+ - pytest
+ - pytest-runner
+ - pytest-xdist
+ - pytest-mock
+ - spyder
+ - pyzmq
+ - wrapt
+ - lxml
+ - scipy
+ - gitpython
+ - pandas
+ - testpath>=0.4.4
+ - tqdm
+ - tabulate
+ - pandoc
+ - contextlib2 # for broadbean
```

Comment:
That comment (and probably the whole line can be removed)

Reply:
Good point.